packages/client/vite.config.ts

```diff
@@ -5,6 +5,13 @@ export default defineConfig({
     fs: {
       strict: false,
     },
+    proxy: {
+      '/rpc': {
+        target: 'http://localhost:8545/',
+        rewrite: () => '/',
+        ws: true,
+      },
+    },
```

Comment:
I've not used this pattern before, and feels a little error prone/easy to miss. Does proxying through vite give us any benefit?

Reply:
Yes it does, as mentioned in the pull request:

> - Add some config to the Vite dev server so that it proxies the Anvil RPC (because the dev server might not be at localhost depending on where the container is run

If you don't proxy requests to the RPC, you have to assume the developer is running the project on the same machine as their browser, which is not the case when using something like Github Codespaces, which is the whole point of this PR 😃

Reply:
Ahh sorry I missed that! That all makes sense.